osu.Game.Rulesets.Mania/UI/Column.cs

```diff
@@ -101,22 +100,25 @@ public Column(int index)
 
         public override Axes RelativeSizeAxes => Axes.Y;
 
-        private bool isSpecial;
+        private ColumnType columnType;
 
-        public bool IsSpecial
+        public ColumnType ColumnType
         {
-            get => isSpecial;
+            get => columnType;
             set
             {
-                if (isSpecial == value)
+                if (columnType == value)
                     return;
 
-                isSpecial = value;
+                columnType = value;
 
-                Width = isSpecial ? special_column_width : COLUMN_WIDTH;
+                AccentColour = columnType.ColumnColour;
+                Width = columnType.DefaultColumnWidth;
             }
         }
 
+        public bool IsSpecial { get; set; }
```

Comment:
Do we even want this to still be a thing? Maybe it shouldn't exist anymore, or at least be a get-only convenience property:
```csharp
public bool IsSpecial => columnType is SpecialColumnType;
```
I'm worried if this survives it'll be bug-prone (two sources of truth for one piece of info).